### ISSUE TITLE
POST /api/menus/:menuId/menu-items Issue

### ISSUE THREAD

**Issue by jordanwillis** [danieloi/Expresso#1]
Before jumping right in, let me say that you did a great job overall on this project!  As I was reviewing the code and testing out the API, I encountered an issue with the below route.

`POST /api/menus/:menuId/menu-items`

I attempted to create a new menu with only the required fields provided (description is the only optional field), however I received a 400 response.  This is just a minor detail, but it looks like you setup your database with description as NOT NULL

https://github.com/danieloi/Expresso/blob/master/migration.js#L30

And subsequently, implemented the API logic to enforce this.

https://github.com/danieloi/Expresso/blob/master/api/menuitems.js#L34

Again, this minor detail is not that big of a deal, but it goes to show how difficult it is to ensure that each aspect of the specification is implemented correctly.  Even the provided test cases did not even check for this scenario.

https://github.com/danieloi/Expresso/blob/master/test/test.js#L1015

Long story short, when you are creating APIs in real life, I highly recommend that you follow a TDD approach and implement tests that correctly validate all corner cases of the API.  Then you don't have to worry about ensuring you have the latest spec in mind when coding the actual API logic.
